src/js/sanitize.js

```diff
@@ -152,9 +152,9 @@ const FRACTIONS = Object.values(FRACT_MAP).join('');
 // Try to match things like "1 tablespoon sugar"
 const RECIPE_QUANTITY_RE = new RegExp([
   '^',
-  `((?:\\d+\\s?)?[\\d${FRACTIONS}⁄-]+)`,
+  `((?:\\.\\d+|\\d+(?:\\.\\d+)?)?(?:\\s*[${FRACTIONS}-])?(?:\\s*⁄\\d+)?)`,
   '\\s*',
-  `(${QUANTITIES.join('|')})?\\.?`,
```

Comment:
Is removing the trailing dot intentional here? iirc it was to handle the occasional trailing punctuation on certain units (`tsp.`, `oz.`)

Reply:
Probably not. I guess I should fix that, and add a test case for this form.